htdocs/main.php

```diff
@@ -174,14 +174,17 @@ function tplFromRequest($param)
     if (isset($caller->controlPanel)) {
         $tpl_data['control_panel'] = $caller->controlPanel;
     }
-
-    if (isset($caller->feedbackPanel) && $user->hasPermission('bvl_feedback')) {
-        $tpl_data['bvl_feedback']   = NDB_BVL_Feedback::bvlFeedbackPossible(
-            $TestName
-        );
-        $tpl_data['feedback_panel'] = $caller->feedbackPanel;
+    if (isset($caller->feedbackPanel)) {
+        if (is_null($user)) {
```

Comment:
@johnsaigle just checking because I'm not entirely sure how `$user` works, but is `is_null` the proper check here? Or should it be `!isset`?

Reply:
Yeah that does look better to me but honestly I'm not that familiar with this part of LORIS. @driusan could you shed some light here?

Reply:
It seems that `is_null()` should work fine, although it will throw a notice if `$user` was never declared as a variable.

Reply:
@HenriRabalais You're right, it would be more appropriate to use `!isset()` rather than `is_null()`. Will update that.